packages/api/config/api_default.yml

```diff
@@ -407,6 +407,10 @@ ApiTokenDefault:
       path: refresh
       method: post
       cors: true
+    - api: backend
+      path: tokenRevoke
+      method: post
```

Comment:
Why a `post` instead of a `delete` 
Like with granules?  https://github.com/nasa/cumulus/blob/master/packages/api/config/api_default.yml#L92

Reply:
This endpoint receives a JWT as a parameter, which is a very long string. I felt like it wasn't ideal to make the route for deleting tokens include the JWT in the path because I was worried about running into URL length limits. 

So I decided to pass the JWT to this route as part of the request body. While I did read that including a request body for DELETE routes is permissible, it felt more awkward to me than using POST. However, I'm happy to use DELETE if others think it makes sense